### DISCUSSION TITLE
add feature on-hover get user detail like image and description

### DISCUSSION THREAD

**Discussion by t-sajid** [afconsult/quill-mention#313] [Q&A] [answered]
if you see the tinymce [mention ](https://www.tiny.cloud/docs/plugins/mentions/) plugin it gives you alot of events to work with if this you can add into this plugin that would be great. 

![tinymce mention plugin](https://thumbs.gfycat.com/SimilarHighlevelDungbeetle-size_restricted.gif)

**Comment by fridaystreet** [comment]
As far as the list goes, you can create a custom render function for each item. Would just be a case of adding a popover on hover to each custom rendered item.

It would be nice however, to be able to also customise the blot, so again you could easily create that effect yourself

**Comment by fridaystreet** [comment, marked as answer]
actually just realised you can customise the blot with blotName option. Just create your own blot with eg 'mycustommentionblot' and it should I assume use that to render
